sound/playwav.h

```diff
@@ -291,7 +300,11 @@ class PlayWav : StateMachine, public AudioStream {
           len_ -= bytes_to_skip;
           start_ = 0.0;
         }
-
+        
+        default_output->print(" length ");
+        default_output->print(length());
+        default_output->println(" [s]");
+        duration_ = length();
```

Comment:
why?
We already have a length() function.

Reply:
Was a remainder of some earlier experiments, removed now. I did not actually use it for my functions.